Ember-FrontEnd/app/styles/app.css

```
@@ -0,0 +1,16 @@
+body{
+
+    display: flex;
+    flex-wrap: wrap;
+
+}
+.page-title{
+    flex: 0 0 100%;
+}
+#bookdiv{
+    align-items: stretch;
+}
+.books{
+    float: left;
+    margin: 5px;
```

I really like the horizontal layout. How do you feel about adding a little more padding between the books? I think the extra space makes it easier on the eyes. It looks great otherwise!

```suggestion
    margin: 25px;
```